pkg/testutils/lint/passes/norawcookies/norawcookies.go

```diff
@@ -0,0 +1,48 @@
+package norawcookies
+
+import (
+	"go/ast"
+
+	"golang.org/x/tools/go/analysis"
+	"golang.org/x/tools/go/analysis/passes/inspect"
+	"golang.org/x/tools/go/ast/inspector"
+)
+
+var Analyzer = &analysis.Analyzer{
+	Name:     "norawcookies",
+	Doc:      `check for correct use of Cookie constructors`,
+	Requires: []*analysis.Analyzer{inspect.Analyzer},
+	Run:      run,
+}
```

Comment:
You've already written it so I wouldn't change it. But I do wonder for a linter like this whether we get most of the benefit out of a git-grep based linter like some of the others in `pkg/testutils/lint_test.go`, which I only mention because I imagine it would be a lot faster.

That said, this lays the groundwork for adding more checks in the future around aspects of cookie creation if we want.

Reply:
I'm just learning about the linter infra so I'll give this a quick shot and see if I get anywhere.

Reply:
git grep linter is way simpler so I'm keeping it. thanks for the pointer.